crmsh/bootstrap.py

```diff
@@ -44,7 +44,7 @@
 RSA_PRIVATE_KEY = "/root/.ssh/id_rsa"
 RSA_PUBLIC_KEY = "/root/.ssh/id_rsa.pub"
 AUTHORIZED_KEYS_FILE = "/root/.ssh/authorized_keys"
-
+SERVICES_STOP_LIST = ["corosync-qdevice.service", "corosync.service", "hawk.service"]
```

Comment:
I hope this won't bring some issue during the backports as `corosync-qdevice.service` doesn't exist in some branches

Reply:
I will notice to remove `corosync-qdevice.service` in other branches

Reply:
we should then check if the service exists on the function name and don't errror. good catch

Reply:
Maybe it's not even needed. The method already checks if the service is available. What happens if it's not? It fails? Or it just continues? If it continues, we would be fine

Reply:
yep agree if it continues is no problem . is just we shouldn't raise exceptions